### ISSUE TITLE
Some changes in code for support Hikview NVR

### ISSUE THREAD

**Issue by Nikolay-Ch** [qb60/hikvision-video-downloader#6]
Hi,

Hikview NVR have another xml-scheme to get files list... I inspect protocol by using Fiddler
I change file camera_sdk.py (replace one URL, change xml-scheme and change get request to post request)
[camera_sdk.zip](https://github.com/qb60/hikvision-video-downloader/files/11480598/camera_sdk.zip)

Regards,
Nikolay

**Comment by qb60** [comment]
@Nikolay-Ch  Hi! Thank you for additions. I'll use your change about POST request.
About XML I'm not sure it's correct. Have you tried using another TrackIDs? Since you renamed 'trackIDList' tag, it isn't used at all for search and is used some default value (101 on my devices).
I use 101 for video files and 103 for photos, it can be different on your NVR.

**Comment by Nikolay-Ch** [comment]
Yes, I'm sure... I've looked at the messages (via Fiddler) that the NVR (DS-N308) sends back. Messages do not contain the **trackIDList** tag, but do contain **trackList**. After replacing the tag, your script started working. Maybe this is true only for HiWatch NVR, maybe it depends on the firmware version, but I would write this in the documentation. If the option with **trackIDList** does not work, the user could change the tag to **trackList**

PS
Stream number (101, 102, 103) - consists of three digits. The first digit are the camera ID on the recorder, and the second and the third digits is the stream number from the camera.

**Comment by qb60** [comment]
Uhh, I see. Seems NVR works different from cameras. Also since NVR captures many cameras, an additional parameter for camera ID is probably needed for the script.
